src/Serilog/Serilog.csproj

```diff
@@ -1,16 +1,17 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>Simple .NET logging with fully-structured events</Description>
-    <VersionPrefix>3.1.2</VersionPrefix>
+    <VersionPrefix>4.0.0</VersionPrefix>
     <Authors>Serilog Contributors</Authors>
+    <!-- .NET Framework version support is frozen at these two TFMs. -->
     <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT'">net462;net471</TargetFrameworks>
-    <TargetFrameworks>$(TargetFrameworks);netstandard2.1;netstandard2.0;net5.0;net6.0;net7.0</TargetFrameworks>
+    <!-- Policy is to support `netstandard2.0`, all active LTS versions, and the latest RTM version. -->
```

Comment:
calling this 'support' is what kicked off this debate about busywork, so language is critical here
though this is a bit long...
```suggestion
    <!-- Policy is to trim TFM-specific builds to `netstandard2.0`, `net6.0` and active LTSes when releasing major versions -->
```

Reply:
Thanks for the review; I'll merge #2015 and address those items here 👍